addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -291,20 +291,22 @@
                         <t t-name="kanban-card">
                             <div class="d-flex flex-row">
                                 <span t-att-title="record.employee_id.value">
-                                    <field name="employee_id" widget="image" options="{'preview_image': 'avatar_128'}" class="o_image_40_cover me-2 float-start"/>
+                                    <field name="employee_id" widget="image" options="{'preview_image': 'avatar_128'}" class="o_image_64_cover me-2 float-start"/>
                                 </span>
                                 <div class="d-flex flex-column text-truncate lh-sm col-10">
-                                    <span class="text-truncate" t-att-title="record.project_id.value"><field name="project_id" class="p-0 fw-bold fs-5"/></span>
-                                    <span class="text-truncate" t-att-title="record.task_id.value"><field name="task_id" class="fst-italic"/></span>
-                                    <span class="text-truncate" t-att-title="record.name.value"><field name="name"/></span>
+                                    <span class="text-truncate" t-att-title="record.project_id.value"><field name="project_id" invisible="context.get('default_project_id')" class="p-0 fw-bold fs-5"/></span>
+                                    <span class="text-truncate" t-att-title="record.task_id.value"><field name="task_id" invisible="context.get('default_project_id')"/></span>
+                                    <span class="text-truncate" t-att-title="record.task_id.value"><field name="task_id" invisible="not context.get('default_project_id')" class="p-0 fw-bold fs-5"/></span>
```

Comment:
To avoid having a span empty. 
```suggestion
                                    <span class="text-truncate" t-att-title="record.task_id.value">
                                        <field name="task_id" invisible="context.get('default_project_id')"/>
                                        <field name="task_id" invisible="not context.get('default_project_id')" class="p-0 fw-bold fs-5"/>
                                    </span>
```